### ISSUE TITLE
Tree file now gives a different output.

### ISSUE THREAD

**Issue by Leitemfa** [metagenome-atlas/atlas#256]
Hi, This may not be a bug, but now the output tree for genomes gives a different tree. Previous version of this file contained a tree for the MAGs but the new version does not. I would like to understand why this changed, and how can I get a tree similar as done before.

Thanks in advance,
Marcio

**Comment by SilasK** [comment]
In the new version #257 you will be able to get a tree by modifying the config file as. 

```
annotations:
  - gtdb_tree
  - gtdb_taxonomy
  - checkm_taxonomy # optional
```

This new version used the GTDB to classify taxonomy and to build a tree for bacteria and archaea (separately). It also uses a correct midpoint rooting. 

I just run the last tests.

Would you need a tree of archeae and bacteria combined?

**Comment by Leitemfa** [comment]
Hi Silas,

Thanks for the quick reply. In fact I would like to have a tree with archaea and bacteria combined. How do I do that?

**Comment by SilasK** [comment]
Do you know how to use git branches?
I added the function again in the branch GTDB.

You would need to add :
```
annotations:
  - checkm_tree
```

**Comment by Leitemfa** [comment]
Thanks, I do not know how to use git branches.

**Comment by SilasK** [comment]
No, problem, I'm working on this. I hope tomorrow or so I will publish the atlas 2.2 release on conda. With all the trees working. You can watch releases on this repo. 
See you.

**Comment by SilasK** [comment]
Fixed by #257
